applicatie/main/routes.py

```diff
@@ -98,22 +99,16 @@ def matrix(jsonbestand, mutatie=None):
     # javascript debug status
     js_debug_status = (current_app.config['ENV'] == 'debugjavascript')
 
-    verwerking = Verwerking(jsonbestand)
+    try:
+        verwerking = Verwerking(jsonbestand)
+    except OSFAanroepError:
```

Comment:
Is het mooier deze fout binnen Verwerking af te handelen?

Reply:
hmm, ik vind van niet, aangezien wat er moet gebeuren bij deze exceptie, is dat er het index scherm met een foutmelding getoond moet worden, dat is dus logica die in routes.py thuishoort: er moet immers een ander scherm getoond worden. Anders moet ik op de een of andere manier in Verwerking dit afvangen en weer doorgeven naar routes.py, lijkt me een extra overbodige stap.